### PR TITLE
Korean translation

### PR DESCRIPTION
**Pull request by fgo0615**
reference #154 
I finished translating into korean

### CODE REVIEW

**Comment by fgo0615** [comment]
Could you please review and merge or comment what sould i do next?

**Review by denysdovhan — COMMENTED** [review]
You should find someone who speaks Korean to read your translation and check for typos and mistakes.

File: README-kr [hunk 1]
```diff
@@ -0,0 +1,1784 @@
+# What the f\*ck JavaScript?
```
Comment (TERADA-DANTE) [inline README-kr:1]: > 아니 X발? 자바스크립트 이게 뭐야??

The title needs to be translated.

File: README-kr [hunk 2]
```diff
@@ -0,0 +1,1784 @@
...
+
+# Node Packaged Manuscript
+
+이 핸드북은 `npm`를 이용하여 설치할 수 있습니다. 그냥 실행합시다:
+
+```
+$ npm install -g wtfjs
+```
+
+이제 당신은 커맨드 창에서 'wtfjs'를 실행할 수 있게 되었습니다. 당신이 선택한 '$PAGER'에서 열릴 것 입니다. 그렇지 않으면 계속 여기서 읽어도 됩니다.
```
Comment (TERADA-DANTE) [inline README-kr:24]: > ~당신이 선택한 '$PAGER'에서 열릴 것 입니다~
> 당신이 선택한 '$PAGER'에서 'wtfjs'가 열릴 것 입니다.

Slightly more elaborated expression.

File: README-kr [hunk 3]
```diff
@@ -0,0 +1,1784 @@
...
+# 번역
+
+현재, **wtfjs**는 아래와 같은 언어로 번역되었습니다.:
+
+- [中文版](./README-zh-cn.md)
+- [Français](./README-fr-fr.md)
+- [Português do Brasil](./README-pt-br.md)
+- [Polski](./README-pl-pl.md)
+- [Italiano](./README-it-it.md)
+
```
Comment (TERADA-DANTE) [inline README-kr:37]: Needs **korean.md** in case of approval

File: README-kr [hunk 4]
```diff
@@ -0,0 +1,1784 @@
...
+[**다른 번역 **][tr-request]
+
+[tr-request]: https://github.com/denysdovhan/wtfjs/issues/new?title=Translation%20Request:%20%5BPlease%20enter%20language%20here%5D&body=I%20am%20able%20to%20translate%20this%20language%20%5Byes/no%5D
+
```
Comment (TERADA-DANTE) [inline README-kr:41]: 38~40 is outdated.
It is different from what the original says

File: README-kr [hunk 5]
```diff
@@ -0,0 +1,1784 @@
...
+<!-- prettier-ignore-start -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+# 목차
+
+- [💪🏻 Motivation](#-motivation)
```
Comment (TERADA-DANTE) [inline README-kr:47]: ~>[💪🏻 Motivation](#-motivation)~
>[💪🏻 시작하기에 앞서](#-motivation)

It needs to be translated.

File: README-kr [hunk 6]
```diff
@@ -0,0 +1,1784 @@
...
+- [✍🏻 Notation](#-notation)
```
Comment (TERADA-DANTE) [inline README-kr:48]: >~[✍🏻 Notation](#-notation)~
>[✍🏻 표기법](#-notation)

It needs to be translated.

File: README-kr [hunk 7]
```diff
@@ -0,0 +1,1784 @@
...
+- [👀 Examples](#-examples)
+  - [`[]`와 `![]`은 같다](#-is-equal-)
+  - [`true`는 `![]`와 같지 않지만, `[]` 이와도 같지 않다](#true-is-not-equal--but-not-equal--too)
+  - [true는 false이다](#true-is-false)
+  - [baNaNa](#banana)
+  - [`NaN`은 `NaN`이 아니다](#nan-is-not-a-nan)
+  - [이것은 실패다](#its-a-fail)
```
Comment (TERADA-DANTE) [inline README-kr:55]: omitted [Object.is() and === weird cases](https://github.com/denysdovhan/wtfjs#objectis-and--weird-cases)

better be
> [Object.is()와 ===에 있어서 이상한 경우들]

File: README-kr [hunk 8]
```diff
@@ -0,0 +1,1784 @@
...
+  - [`[]`은 truthy이지만 `true`은 아니다](#-is-truthy-but-not-true)
+  - [`null`은 falsy이지만 `false`은 아니다](#null-is-falsy-but-not-false)
+  - [`document.all`은 객체이지만 정의되지 않았다](#documentall-is-an-object-but-it-is-undefined)
```
Comment (TERADA-DANTE) [inline README-kr:58]: > ~[`document.all`은 객체이지만 정의되지 않았다](#documentall-is-an-object-but-it-is-undefined)~
> [`document.all`은 객체이지만 `undefined`이다](#documentall-is-an-object-but-it-is-undefined)

Slightly more Intuitive

**Comment by TERADA-DANTE** [comment]
@denysdovhan  Good day!
I found your fabulous repo and decide to contribute.
FYI, I am a native speaker of korean and japanese.

Thanks to @fgo0615 , for his amazing translation on korean, I've check it and guarantee you that It deserves to be merged.

Still it has some missings(not typo error, just omitted due to main branch update.), I'd like to add some to recover. In this case, Do I still need a cover-up man to verify it?

**Comment by denysdovhan** [comment]
@fgo0615 please, apply suggestions from @TERADA-DANTE.

@TERADA-DANTE please, collaborate with @fgo0615 on adding new examples.

**Comment by denysdovhan** [comment]
Already translated by #201

**Pull request closed without merging**